applications/dashboard/controllers/api/MediaApiController.php

```diff
@@ -0,0 +1,51 @@
+<?php
+/**
+ * @copyright 2009-2018 Vanilla Forums Inc.
+ * @license GPLv2
```

Comment:
https://github.com/vanilla/porter/pull/151

GPLv2 is invalid. It should be GPL-2.0

Just an FYI